src/cdk/aspects/security/policies.ts

```diff
@@ -22,6 +29,8 @@ export class Policies implements cdk.IAspect {
      *
      * https://docs.aws.amazon.com/IAM/latest/UserGuide/list_amazonmobileanalytics.html
      * https://github.com/aws/aws-cdk/blob/master/packages/%40aws-cdk/aws-certificatemanager/lib/dns-validated-certificate.ts
+     * @internal
+     * @ignore
```

Comment:
The user can choose to display only public artifacts in the docs UI - is that better than tagging all the private ones to hide them? Could we make 'public' the default?

Reply:
Good suggestion, I will see what can be done with typedoc and try and make this cleaner.

Reply:
As discussed, we'll go with this for now